sdk/translation/Azure.AI.Translation.Document/tests/samples/README.md

```diff
@@ -0,0 +1,3 @@
+Source files in this directory are written as tests from which samples are extracted.
```

Comment:
Proliferating copies of the same text could result in drift if someone changes one and they get out of sync ... is there a way to practice DRY for this README at all?

Reply:
In order to show in all relevant directories, they need to actually exist in those directories. Even if text drifts a little, the message is only "these are not the samples you're looking for" with a link to the sample sources.